lib/watir-webdriver/locators/element/locator.rb

```diff
@@ -0,0 +1,249 @@
+module Watir
+  module Locators
+    class Element
+      class Locator
+        attr_reader :selector_builder
+        attr_reader :element_validator
+
+        WD_FINDERS = [
```

Comment:
What do you think about using the [constant defined in Selenium](https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/webdriver/common/search_context.rb#L25)?